tests/Provide/Error/ThrowableHandlerTest.php

```
@@ -41,7 +41,7 @@ public function testError(): void
         $e = null;
         try {
             echo HELLO; // @phpstan-ignore-line
-        } catch (Throwable $e) {  // @phpstan-ignore-line create $e
+        } catch (Throwable $e) { // @phpstan-ignore catch.neverThrown
```

PHPStan (2.2.7)

[error] No error with identifier catch.neverThrown is reported on line 44.


(ignore.unmatchedIdentifier)